hardware/EcoDevices.cpp

```diff
@@ -286,7 +287,7 @@ void CEcoDevices::GetMeterDetails()
 	else
 	{
 		message = "EcoDevices firmware needs to be at least version ";
-		message = message + static_cast<std::ostringstream*>(&(std::ostringstream() << min_major << "." << min_minor << "." << min_release))->str();
+		message = message + boost::lexical_cast<std::string>( min_major ) + "." + boost::lexical_cast<std::string>(min_minor) + "." + boost::lexical_cast<std::string>(min_release);
```

Comment:
Whats wrong with stringstream ? not a fan of depending on lexical cast

Reply:
After enabling cpp11 LLVM on Mac fails to build with erros on those lines. Didn't save the error message, but I would guess it's about taking the address of a temporary. I guess there is no reason not to use operator . instead of operator ->

We are already depending on lexical_cast in other places btw.

Reply:
Yes and i hope to remove the dependencies soon...
I am all for c++11 as long as raspberry pi users for instance are able to build as well